backend/src/oe/api/routes.ts

```diff
@@ -212,6 +213,29 @@ class OpEnergyRoutes {
     }
     logger.info( `${UUID}: PROFILE: end: $getBlockByHash`);
   }
+
+  private async $getBlockSpanList(req: Request, res: Response) {
+    const UUID = await opEnergyApiService.$generateRandomHash();
+    try {
+      logger.info( `${UUID}: PROFILE: start: $getBlockSpanList`);
+      const { startBlockHeight, span, numberOfSpan } = req.params;
+
+      if (
+        !isValidPositiveNumber(startBlockHeight) ||
+        !isValidPositiveNumber(span) ||
+        !isValidPositiveNumber(numberOfSpan)
```

Comment:
I agree, that `numberOfSpan` and `span` should be positive, but `startBlockHeight` should be Natural, as we can ask to build spans starting from 0

Reply:
Done